src/test/java/seedu/address/logic/commands/ListCommandTest.java

```diff
@@ -26,14 +29,28 @@ public void setUp() {
         expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
     }
 
+
     @Test
     public void execute_listIsNotFiltered_showsSameList() {
-        assertCommandSuccess(new ListCommand(), model, ListCommand.MESSAGE_SUCCESS, expectedModel);
+        ListCommand command = null;
+        try {
+            command = new ListCommandParser().parse("");
```

Comment:
Could I ask if the `ListCommandParser` is used here? 
Uncertained as I think that this is a [Unit Testing](https://nus-cs2103-ay2223s2.github.io/website/schedule/week3/topics.html#w3-7-unit-testing) and this test file is to test for correctness of `ListCommand` class. For `ListCommandParser` class, is it better to create a separate test file named `ListCommandParserTest` and implement some test case based on the method in `ListCommandParser` class?
If I had some misunderstanding about your code, you could always correct me.

Reply:
Ah yes, you are right. I've changed it, do take a look, thanks for the suggestion!